filebeat/Dockerfile

```diff
@@ -4,7 +4,7 @@ MAINTAINER Nicolas Ruflin <ruflin@elastic.co>
 RUN set -x && \
     apt-get update && \
     apt-get install -y --no-install-recommends \
-         netcat python-pip virtualenv && \
+         netcat python-pip rsync virtualenv && \
```

Comment:
`rsync` added but not used?

Reply:
Yes and now. It was used when `make update` was called inside the container. Now it happens outside again. The reason I would like to keep it in is that if someone wants to tests things from inside the container and run `make update` rsync will be needed for Filebeat.